apps/swirl-docs/src/styles/globals.css

```diff
@@ -48,15 +48,19 @@ body {
 }
 
 h1 {
-  font-size: var(--s-font-size-2xl);
+  font-size: 36px;
```

Comment:
**issue:** Let's use relative units to not break browser zoom / font size settings. Applies to the other pixel values as well.

Reply:
yeah good point. my fault. started with rem and then changed to px unnecessarilily